superbench/runner/runner.py

```diff
@@ -199,6 +199,24 @@ def deploy(self):    # pragma: no cover
             )
         self._ansible_client.run(self._ansible_client.get_playbook_config('deploy.yaml', extravars=extravars))
 
+    def run_sys_info(self):
+        """Run the system info on all nodes."""
+        self.check_env()
+
+        logger.info('Runner is going to get info.')
```

Comment:
`Runner is going to get node system info`

Reply:
updated, thanks